models/base.models.js

```diff
@@ -0,0 +1,6 @@
+const fs = require('fs/promises')
+
+exports.selectEndpointsInfo = () => {
+    return fs.readFile(`./endpoints.json`)
+        .then(endpoints => JSON.parse(endpoints))
+}
```

Comment:
We don't actually need a model for this task. Everything can be done in the controller. 🙂